Added new defintions (#53)
lhcb/glossary: glossary/P.md

Graphical display of detector and event information. [Project website](http://lhcbdoc.web.cern.ch/lhcbdoc/panoramix/).

## PCIe40: PCIexpress {#PCIe40}

> «Readout boards:ReadoutBoard» used for the «data acquisition system:DAQ», «Experiment control system:ECS» and «timing and fast control:TFC», see [LHCb Trigger and Online TDR](https://cds.cern.ch/record/1701361/files/LHCB-TDR-016.pdf).

Reviewer: ```suggestion
«Readout boards:ReadoutBoard» used for the «data acquisition system:DAQ», «Experiment Control System:ECS» and «Timing and Fast Control:TFC». Refer to the [LHCb Trigger and Online TDR](https://cds.cern.ch/record/1701361/files/LHCB-TDR-016.pdf) for details.
```

> Each board has 48 bidirectional optical links for interfacing with frontend electronics and one bidirectional optical link for interfacing the TFC.

Reviewer: ```suggestion
Each board has 48 bidirectional optical links for interfacing with frontend electronics and one bidirectional optical link for interfacing the «TDC».
```

> Each board has 48 bidirectional optical links for interfacing with frontend electronics and one bidirectional optical link for interfacing the «TDC».

Reviewer: TDC should be TFC, right ;-)?

> Each board has 48 bidirectional optical links for interfacing with frontend electronics and one bidirectional optical link for interfacing the «TDC».

Reviewer: ```suggestion
Each board has 48 bidirectional optical links for interfacing with the frontend electronics and one bidirectional optical link for interfacing with the «TFC».
```
